tests/test_multiplexer.py

```diff
@@ -75,14 +75,25 @@ def test_connect_twice():
     multiplexer.disconnect()
 
 
+def test_disconnect_twice():
+    """Test that connecting twice the multiplexer behaves correctly."""
+    multiplexer = Multiplexer([_make_dummy_connection()])
+
+    assert not multiplexer.connection_status.is_connected
+    multiplexer.connect()
+    assert multiplexer.connection_status.is_connected
+    multiplexer.disconnect()
+    multiplexer.disconnect()
```

Comment:
disconnect twice

Reply:
this is the idea, check double disconnect. test named disconnect twice)